### PR TITLE
Ease listing contributors

### PR DESCRIPTION
**Pull request by clementbiron**
Install and set up @all-contributors on this repository.

### CODE REVIEW

File: .all-contributors.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+This list of contributors is generated automatically by [All Contributors](https://allcontributors.org/). The output used in production is the JSON format, this file is not used.
```
Comment (Ndpnt) [inline .all-contributors.md:1]: Add a comment to explain that this file is needed can not be removed

File: .all-contributorsrc [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "projectName": "engine",
+  "projectOwner": "OpenTermsArchive",
+  "files": [
+    ".all-contributors.md"
+  ],
+  "contributors": [],
+  "repoType": "github",
+  "commitConvention": "angular",
```
Comment (Ndpnt) [inline .all-contributorsrc:9]:
```suggestion
  "commitConvention": "none",
```

File: .all-contributorsrc [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  "commitType": "docs",
```
Comment (Ndpnt) [inline .all-contributorsrc:10]: I do not see the `commitType` configuration option in the doc

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,16 @@
 
 All changes that impact users of this module are documented in this file, in the [Common Changelog](https://common-changelog.org) format with some additional specifications defined in the CONTRIBUTING file. This codebase adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased [no-release]
```
Comment (Ndpnt) [inline CHANGELOG.md:5]: When you consider that a release is not needed you have to follow [these guidelines](https://github.com/OpenTermsArchive/engine/blob/main/CONTRIBUTING.md#changelog)

File: package.json [hunk 5]
```diff
@@ -103,6 +103,7 @@
   },
   "devDependencies": {
     "@commitlint/cli": "^19.0.3",
+    "all-contributors-cli": "^6.26.1",
```
Comment (Ndpnt) [inline package.json:106]: It seems that this dependency is not [required anymore](https://github.com/all-contributors/cli). Am I wrong?

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -180,3 +180,11 @@ In contrast, we consider errors from the `recorder` module as fatal, and we cras
 **The best way to handle programmer errors is to crash immediately.** Indeed, it is not recommended to attempt to recover from programmer errors — that is, allow the current operation to fail, but keep handling requests. Consider that a programmer error is a case that you didn’t think about when you wrote the original code. How can you be sure that the problem won’t affect the program itself and the data integrity?
 
 This section is highly inspired, and in part extracted, from [this error handling guide](https://console.joyent.com/node-js/production/design/errors).
+
+## List a new contributor in the Open Terms Archive website
+
+We acknowledge the efforts of our contributors by listing them on our [website](https://opentermsarchive.org) and this is made possible by the use of the [All Contributors bot](https://allcontributors.org/docs/en/bot/overview).
+
+That enables adding a contributor with a comment on an issue or pull request, without writing code. To do this, please use the [dedicated issue](https://github.com/OpenTermsArchive/engine/issues/1073) on this repository.
```
Comment (Ndpnt) [inline CONTRIBUTING.md:188]:
```suggestion
All Contributors enables adding a contributor with a comment on an issue or pull request, without writing code. To do this, please use the [dedicated issue](https://github.com/OpenTermsArchive/engine/issues/1073) on this repository.
```

File: package.json (hunk 5 above)
Comment (clementbiron) [reply package.json]: I've removed it.

File: .all-contributors.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+This file is not used, but it is mandatory and cannot be deleted because it is generated automatically by [All Contributors](https://allcontributors.org/) to list the contributors. The output used in production is in JSON format and is located in the `.all-contributorsrc` file.
```
[the remark this answers is not in the document]
Comment (clementbiron) [reply .all-contributors.md:1]: I think we can't ignore because this file have to be exposed on this repository and could be updated by using the CLI.

Comment (Ndpnt) [inline .all-contributors.md:1]:
```suggestion
This file is required by [All Contributors](https://allcontributors.org/) to work properly and must not be deleted. Although it is automatically updated with the list of contributors, the actual output used in production is in JSON format and located in the `.all-contributorsrc` file.
```

**Review by Ndpnt — APPROVED** [review]

**Pull request merged**